src/sentry/integrations/vercel/integration.py

```diff
@@ -107,48 +142,57 @@ def get_organization_config(self):
                 "type": "project_mapper",
                 "mappedDropdown": {
                     "items": vercel_projects,
-                    "placeholder": "Select a Vercel Project",  # TOOD: add translation
+                    "placeholder": _("Choose Vercel project..."),
```

Comment:
```suggestion
                    "placeholder": _("Choose Vercel project\u2026"),
```
We should use ellipsis instead of `...`.

Reply:
@markstory  This change causes `\u2026` to actually show up in the visible text (but the other one worked btw)